src/workerProcess.ts

```diff
@@ -7,44 +7,53 @@ import { Policy } from "./Policy";
 import { readFile } from "fs";
 import { FileMatcher } from "./FileMatcher";
 
-
 export type Message =
-  | { type: "match", policyName: string, match: Match }
-  | { type: "processedFile", filePath: string }
-
+  | { type: "match"; policyName: string; match: Match }
+  | { type: "processedFile"; filePath: string };
 
 export function processFile(
   filePath: string,
   cwd: string,
   config: Config,
   onMatch: (match: Match, policy: Policy) => void,
-  onDone: (filePath: string) => void,
+  onDone: (filePath: string) => void
 ) {
   readFile(join(cwd, filePath), { encoding: "utf8" }, (err, fileContents) => {
```

Comment:
Could potentially update this to fs/promises now as well but the callback pattern is already here in the workerProcess, so 🤷🏻

Reply:
Not going to do this now, could come as a separate change if desired.